lib/rental.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'stdio'
+
+class Rental
+  attr_accessor :date, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.rent_book(self)
+    person.book_rents(self)
+  end
+
+  def self.create(book:, student:, teacher:)
+    people = student.all + teacher.all
+    book_index, person_index, date = StandardIO.rental_prompt(book: book, people: people)
+
+    person = people[person_index]
+    book = Book.all[book_index]
```

Comment:
- [ ] You should follow the dependency inversion principle. Your classes should not depend on another module or class. You should pass the name of the book instead of passing the index and depending on the Book class to get the name.